docs/api/trigger_run_parameters.md

```diff
@@ -19,34 +20,98 @@ So the job configurations are typically not a suitable model for end users to sp
-
-When Orchestrator receives a request to trigger an ORT run, before starting the Analyzer step, it invokes a special worker for validating and processing the run parameters provided by the user. This worker loads the job configurations and additional parameters for the current ORT run from the database. It loads a file with the current parameter configuration. This is basically a Kotlin script which takes the parameters provided by the user as input and returns processed parameters and a collection with issues (if any) as output. The script can now perform mapping and validation steps, such as:
-
-* check whether certain parameters have a defined set of values; e.g. only rule sets "standard", "web", "special" are allowed
-* map the values given by the user to other values; e.g. the rule set "standard" could be mapped to a specific configuration file path from where it can be downloaded
-* create specific job configurations for parameters; e.g. the users can specify a set of FossID parameters, which are then added to the corresponding options in the scanner configuration
-
-The following diagram shows a concrete example of such a mapping:
-
-[plantuml,parameters_example,svg]
-----
-include::config_validation_example.puml[]
-----
+What is basically needed is a way to customize the rather technical API of ORT Server towards the needs and capabilities of the environment in which it is running, so that it can be used from clients in a convenient and safe way.
+
+## Parameter Configuration
+
+To address these issues, ORT Server uses a validation and mapping approach of run parameters based on a scripting solution. The idea is as follows:
+
+When Orchestrator receives a request to trigger an ORT run, before starting the Analyzer step, it invokes a special worker for validating and processing the run parameters provided by the user. This worker loads the job configurations and additional parameters for the current ORT run from the database. It loads a file with the current parameter configuration. This is basically a Kotlin script that takes the parameters provided by the user as input and returns processed parameters and a collection of issues (if any) as output. The script can now perform mapping and validation steps, such as:
+
+* check whether certain parameters have a defined set of values; e.g., only rule sets "standard", "web", "special" are allowed
+* map the values given by the user to other values; e.g., the rule set "standard" could be mapped to a specific configuration file path from where it can be downloaded
+* create specific job configurations for parameters; e.g., the users can specify a set of FossID parameters, which are then added to the corresponding options in the scanner configuration
+
+For example, a validation script could use the parameters to configure the scanner and evaluator jobs.
+
+The following input:
```

Comment:
Is this now the replacement for the `config_validation_example.puml`? According to the commit message, I would have expected a Mermaid diagram.

Reply:
Yes, I have clarified it in the commit message now. It makes more sense to me to show the job configurations in JSON like when using the API instead of using a diagram.